robotbase/src/main/java/com/team2357/lib/subsystems/drive/SkidSteerDriveSubsystem.java

```diff
@@ -137,7 +137,12 @@ public final void driveVelocity(double speedInchesPerSecond, double turnDegreesP
 	public void setProportional(double leftProportion, double rightProportion) {
 		m_leftControllers.set(leftProportion);
 		m_rightControllers.set(rightProportion);
-	};
+	}
+
+	public void setSpeed(double speed) {
```

Comment:
Do we need this function? Can't we just call `setProportional(speed, speed)` which would do the same thing?

Reply:
Correct.